src/js/pouch-express.js

```diff
@@ -0,0 +1,288 @@
+/*
+
+    Provide a CouchDB-like API using `PouchDB` and `express-pouchdb`.  This in only useful in association with an
+    existing `gpii.express` instance.  See the documentation for details:
+
+    https://github.com/GPII/gpii-pouchdb/blob/master/docs/pouch-express.md
+
+ */
+"use strict";
+
+/*
+
+    TODO:  Work through the express-pouchdb setup and code with Antranig, as we see many warnings like:
```

Comment:
I imagine this is a bug in express-pouch, given the insufferable bureaucracy of bluebird-promise. It seems that one way we could trace it down is to patch inside express-pouch to include var Promise = require("./js/debug/bluebird"); as per https://github.com/petkaantonov/bluebird/issues/1205 and see where it is.